pkg/model/resources/nodeup.go

```diff
@@ -53,12 +53,9 @@ NODEUP_HASH={{ NodeUpSourceHash }}
 {{ ProxyEnv }}
 
 function ensure-install-dir() {
-  INSTALL_DIR="/var/cache/kubernetes-install"
-  # On ContainerOS, we install to /var/lib/toolbox install (because of noexec)
-  if [[ -d /var/lib/toolbox ]]; then
-    INSTALL_DIR="/var/lib/toolbox/kubernetes-install"
-  fi
-  mkdir -p ${INSTALL_DIR}
+  INSTALL_DIR="/opt/kops"
+  mkdir -p ${INSTALL_DIR}/bin
+  mkdir -p ${INSTALL_DIR}/conf
```

Comment:
Thought: If we're making our directories more standard, `/etc/kubernetes/kops` might be better than `/opt/kops/conf`

Reply:
Usually things that are in /opt have pretty much everything there, with their own structure, /etc seems more like a place for packages that respect the linux structure.
If you like /etc more, maybe `/etc/kops` would be quite nice. Do you think the `kubernetes` part adds any value?

Reply:
OK - the "self contained" argument sounds reasonable in this case :-)  We can consider /etc/kops or /etc/kubernetes/kops for the future (I don't have a strong preference either way on how important having kubernetes in the path is!)